apps/dot-voting/app/components/VoteCardGroup.js

```diff
@@ -47,7 +42,7 @@ const VoteCardGroup = ({ title, count, children }) => {
           <Tag limitDigits={4} label={count} size="small" />
         </span>
       </h2>
-      <CardLayout columnWidthMin={30 * GU} rowHeight={rowHeight}>
+      <CardLayout columnWidthMin={30 * GU} rowHeight={null}>
```

Comment:
I would better omit the prop, or pass an ugly `undefined` instead, since it can cause some issue when used in the tagged template literal [here](https://github.com/aragon/aragon-ui/blob/4dac5c74736e5542a626eb19949d9214b504d3dd/src/components/CardLayout/CardLayout.js#L22), because of the resulting `nullpx`

Omitting it will probably be a better idea as the component will use the defaultProp value instead `21 * GU`
```suggestion
      <CardLayout columnWidthMin={30 * GU}>
```

Reply:
We do _not_ want the height to be `21 * GU`! We want the height to be `"auto"` or `"initial"`. Maybe we can use `css="grid-auto-rows: auto"` instead, though.

I've [started a conversation](https://github.com/aragon/aragon-ui/commit/b2d1da7f7cce7ed3dd7cf843296c41ac13f67100#r35598639) about the `21 * GU` default; please chime in there if you have ideas around how to improve this component!

Reply:
I understand, anyway passing `null` as the prop value does not seem a good solution for that problem, it could also break propTypes in the component probably.